packages/dashboard/components/company-logo.tsx

```diff
@@ -7,10 +7,11 @@ import { useRouter } from 'next/router';
 export const CompanyLogo = ({ sx }: any) => {
   const theme = useTheme();
   const router = useRouter();
+
   const logo =
     theme.palette.mode === 'dark'
-      ? '/assets/logo-with-text.white.svg'
-      : '/assets/logo-with-text.purple.svg';
+      ? process.env.LOGO_LIGHT || '/assets/logo-with-text.white.svg'
```

Comment:
@ipbyrne for a front ent component like this, how do we safely pull these UI values from a config, without accidentally pulling secrets?

Reply:
Looks like it might need to be the other way around. I might need to prepend `NEXT_PUBLIC_` onto the environment variable. 

https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser

Reply:
you should be importing from a config, and passing props through from the page component... see https://github.com/transmute-industries/api.did.actor/blob/main/pages/%5Bdid%5D/index.tsx#L9

Don't use process.env anywhere but in the config, and definitely don't expose that config to the client without explicitly mapping the safe for the client values through.

Reply:
Hey @BenjaminMoe and @OR13 !

That is correct in that if we append `NEXT_PUBLIC_` to the environment variable name, it will be able to be safely loaded to the client-side of the application.